src/components/notes.js

```diff
@@ -0,0 +1,47 @@
+import "../App.css";
+import { useState } from "react";
+
+const Note = () => {
+  const [show, setShow] = useState(false);
```

Comment:
Try to give this variable name more descriptive. I didn't strike immediately. I think you can use displayStatus or todoDisplayStatus

Reply:
done